services/parser-database/connector.py

```diff
@@ -0,0 +1,70 @@
+"""conector.py - One stop connector for external
+services/databases"""
+import os
+import requests  # pylint: disable=import-error
+
+
+articles_in_memory = {}
+keywords_in_memory = {}
+
+
+def get_documents_to_parse():
+    # When database is integrated, this will go away
+    mty_document = {
+        "hash":
+        "afafbfbdce8c40924edae00f6ce54f0c639ce42a2" +
+        "c0fbbfa6ab82ea6925827c51",
+        "jurisdiction":
+        "Monterrey",
+        "url":
+        "http://www.guadalupe.gob.mx/wp-content/up" +
+        "loads/2019/09/Nuevo-Reglamento-Homologado-1.pdf",
+    }
+    document_list = []
+    document_list.append(mty_document)
+    return document_list
+
+
+def get_keywords(text):
+    """Get keywords that relate to this article (Javier's service)
+
+    Args:
+        text (sting): text to extract keywords from
+
+    Returns:
+        [list]: list of extracted keywords
+    """
+
+    extracted_keywords = []
+    request = {'text': text}
+    nlp_output = requests.post(os.getenv("KEYWORDS_SERVICE"), json=request)
+    # nlp_output = keywordmock.get_keywords(text)
+    json_output = nlp_output.json()
```

Comment:
It seems that this would break if the response is not in JSON format (i.e., Javier's service returns an error as response). Probably it would be worth it to wrap this in a `try/except` block.